public/components/common/search/direct_search.tsx

```diff
@@ -19,7 +19,8 @@ import {
   EuiToolTip,
 } from '@elastic/eui';
 import { i18n } from '@osd/i18n';
-import { isEmpty, isEqual } from 'lodash';
+import isEmpty from 'lodash/isEmpty';
+import isEqual from 'lodash/isEqual';
```

Comment:
can https://webpack.js.org/configuration/optimization/#optimizationusedexports help to tree shake the
```tsx
import { isEmpty, isEqual } from 'lodash';
```

or maybe `lodash-es` since it's required by core 
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/f728b5ae92de732bcf0f491ecf4f90f987291593/yarn.lock#L12596

```tsx
import { isEmpty, isEqual } from 'lodash-es';
```

I'm asking because seems devs need to manually type `import isEmpty from 'lodash/isEmpty';` as tsserver doesn't suggest default exports. someone using lodash could again introduce `import { xxx } from 'lodash';` and bring in the whole library